network/loop_test.go

```
@@ -164,18 +164,6 @@ func TestTLSConfigValidation(t *testing.T) {
 			},
 			wantLoop: true,
 		},
-		{
-			name: "Partial TLS config (cert only)",
-			tlsConfig: types.ConnectionConfig{
-				URL:           "https://example.com",
-				TLSCert:       "cert",
-				BatchCount:    10,
-				FlushInterval: time.Second,
-				Timeout:       time.Second,
-				UserAgent:     "test-client",
-			},
-			wantLoop: true,
-		},
```

why is this test deleted?

---

The prometheus client actually handles certificates much better for validation so passing in dummy data fails.